### ISSUE TITLE
Readrops asks for too much articles, overloads the server and no article is retrieved

### ISSUE THREAD

**Issue by Massedil** [readrops/Readrops#116]
I wanted to test your app with my FreshRSS account but I can't use it because it seems that the app request too much articles in one request. My server crashes with a `PHP Fatal error:  Allowed memory size of 134217728 bytes exhausted`.

Logs : 

```
"GET /api/greader.php/reader/api/0/tag/list?output=json HTTP/1.1" 200 7852 "-" "okhttp/4.9.1"
"GET /api/greader.php/reader/api/0/subscription/list?output=json HTTP/1.1" 200 151146 "-" "okhttp/4.9.1"
"GET /api/greader.php/reader/api/0/stream/contents/user/-/state/com.google/reading-list?xt=user%2F-%2Fstate%2Fcom.google%2Fread&xt=user%2F-%2Fstate%2Fcom.google%2Fstarred&n=5000 HTTP/1.1" 500 240 "-" "okhttp/4.9.1"
```

[I reported the problem here](https://github.com/FreshRSS/FreshRSS/issues/3736) and it seems that it would be better to request less articles in the retrieve request. 

@Frenzie pointed [an helpful comment here](https://github.com/FreshRSS/FreshRSS/issues/2566#issuecomment-541317776).

**Comment by Shinokuni** [comment]
Indeed, Readrops fetches 5000 unread items at initial synchronization, which can be too much for some servers. I set this number for two reasons : 
- I would like to implement local search and thus be able to search among a lot of items
- With such a number of items, there was no reason to add any kind of pagination

With fetching like only 1000 thousands items, there is a risk that some feeds won't have any unread items and Readrops currently can't fetch items by feed or do any kind of pagination.

**Comment by Daniel15** [comment]
Try increase the memory limit in php.ini

**Comment by Shinokuni** [comment]
Readrops fetch limit is now 2500 articles instead of 5000, so it should less hurt your php instance.
